README.md

```diff
@@ -1 +1,43 @@
-# Leaderboard
\ No newline at end of file
+# Leaderboard
+
+This project was built to display a leaderboard that displays scores submitted by different players. This project was built using webpack and will be served using webpack dev server.
+
+> This is a project containing several file such as: 
+1. index.html
+2. style.css
+3. index.js
+4. scores.js
+5. show.js
+
+## Built With
+
+-- HTML, CSS, JavaScript
+-- Used Webpack as bundler
+-- (Used Lighthouse, Webhint, Stylelint, and ESLint as linters)
+
+
+## Getting Started
+This project makes use of Webpack to bundle all the JS modules.
+
+Clone the repository into your local machine and then run npm install to download all the dependencies. Then after making any changes inside the src/ folder, you can run npm run build to generate the corresponding files inside dist/ directory.
+
+Run tests
+If you want to test your changes on the fly without having to build every time, you can use npm run start to start the webpack-dev-server in development mode. This will watch for any changes you make and update the page accordingly.
+
+To check the linters on your local env, make sure you have installed all dependencies (npm install) and then copy and paste the following files inside the root of the project: .hintrc, .stylelintrc.json, and .eslintrc.json.
+
+### Prerequisites
+    You only need a good computer with a browser and VS Code.
+
+## Authors
+
+👤 **Abdul Rahman Daudzai**
+
+- GitHub: [@wadaudzai](https://github.com/wadaudzai)
+- Twitter: [@wadaudzai](https://twitter.com/wadaudzai)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abdulrahman-daudzai-490421234/)
+
+
+## Show your support
+
```

Comment:
- [ ] **Optional:** In future projects, add the Microverse given `MIT` license and a few snippets in sections like getting started, for example, you could include the snippet to clone the repo `git clone [repo_link]`. This gives your repo a professional touch.